docs/types/value-types.rst

```diff
@@ -300,6 +300,9 @@ All three functions ``call``, ``delegatecall`` and ``staticcall`` are very low-l
 The ``gas`` option is available on all three methods, while the ``value`` option is not
 supported for ``delegatecall``.
 
+.. note::
+    It is best to avoid relying on hardcoded gas values in your smart contract code, regardless of whether state is read from     or written to, as this can have many pitfalls. Also, access to gas might change in the future.
```

Comment:
Nice! Can you add a line break at roughly 90 characters?

Reply:
Ok done. Sorry for the bad edit.